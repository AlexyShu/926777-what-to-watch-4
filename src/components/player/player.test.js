import React from "react";
import renderer from "react-test-renderer";
import Player from "./player.jsx";

const mockfilm =
{
  name: ``,
  video: ``,
  genre: ``,
  posterSrc: ``,
  year: 2015,
  ratingScore: ``,
  ratingLevel: ``,
  ratingCount: ``,
  descriptionPartOne: ``,
  descriptionPartTwo: ``,
  filmDirector: ``,
  filmStarring: ``,
};

const mockBool = false;

describe(`Render correct Player`, () => {
  it(`Render Player`, () => {
    const tree = renderer
    .create(<Player
      film = {mockfilm}
      autoPlay = {mockBool}
      muted = {mockBool}
    />)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
