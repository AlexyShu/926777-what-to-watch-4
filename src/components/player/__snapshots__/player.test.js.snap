// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correct Player Render Player 1`] = `
<video
  controls={true}
  poster=""
  width="100%"
>
  <source
    src=""
  />
</video>
`;
